pkg/schema/tracker.go

```diff
@@ -131,13 +128,19 @@ func NewTracker(ctx context.Context, task string, sessionCfg map[string]string,
 	}
 
 	for k, v := range sessionCfg {
-		err = se.GetSessionVars().SetSystemVar(k, v)
+		err = se.GetSessionVars().SetSystemVarWithRelaxedValidation(k, v)
 		if err != nil {
+			// when user set some unsupported variable, we just ignore it
+			// tidb error message: https://github.com/pingcap/tidb/blob/f570e8fb679df0686a4023333333e046547b7ff1/errors.toml#L1934
+			if strings.HasPrefix(err.Error(), "[variable:1193]Unknown system variable") {
```

Comment:
why don't use structured error like `ErrUnknownSystemVar`?

Reply:
I forgot that we can access the variables in tidb directly 😂